src/molecules/Flash/Flash.tsx

```diff
@@ -24,14 +24,19 @@ export const Flash: React.FC<FlashProps> = ({
   showIcon,
   dismissible = true,
   autoDismiss = true,
-  duration = 5000,
+  duration,
   ...otherProps
 }) => {
   const [dismissed, setDismissed] = useState(false);
   const [timer, setTimer] = useState(null);
-  const dismiss = () => {
-    setDismissed(true);
-  };
+  const ref = useRef<HTMLDivElement>(null);
+  const dismiss = () => setDismissed(true);
+
+  useEffect(() => {
+    if (!duration && ref.current && ref.current.innerText) {
+      duration = (1000 * ref.current.innerText.split(/\s+/).length) / 3;
+    }
```

Comment:
@tonytrucco-CS let me know if there should be a minimum duration. It's possible that the flash could contain only html elements (even if it's rare) in which the flash would immediately disappear.

```suggestion
    const minDefaultDuration = 3000; // or whatever
    if (!duration && ref.current && ref.current.innerText) {
      duration = Math.max((1000 * ref.current.innerText.split(/\s+/).length) / 3, minDefaultDuration);
    }
```

Reply:
Yeah, let's do 3 seconds as the minimum.

Reply:
Oops GitHub didn't show your comment but it did show the approval so I went ahead and merged. I'll add this on Monday.